templates/report_template.html

```diff
@@ -83,6 +86,7 @@
 							{% else %}
 								<td style="text-align: center;">{{row.lcb_num}}</td>
 							{% endif %}
+							<td style="text-align: center;">{{row.build_days_ago}}</td>
```

Comment:
This case doesn't actually seem to be handled, from what I've seen in the code. What is the behavior if no last build can be found? Have we tested this case?

Reply:
If I understand correctly - if there is no build it will be skipped.

Reply:
@agarcia64 Did you test this scenario when you tested the PR?

Reply:
I didn't, but I will test it now. Will report back with results!

Reply:
After testing with jobs that have no previously available jobs, an error is thrown. 
```
Traceback (most recent call last):
  File "./jeeves.py", line 67, in <module>
    run_report(config, blockers, server, header, test_email, no_email, template_file)
  File "/Users/alfredogarcia/Documents/Github/jeeves/report.py", line 60, in run_report
    jenkins_api_info = get_jenkins_job_info(server, job_name)
  File "/Users/alfredogarcia/Documents/Github/jeeves/functions.py", line 177, in get_jenkins_job_info
    'build_days_ago': build_days_ago
UnboundLocalError: local variable 'build_days_ago' referenced before assignment
```

Reply:
@agarcia64 This comment should go in a new review